Frontend-v1-Original/components/liquidityPairs/LiquidityPairsTable.tsx

```diff
@@ -220,25 +222,27 @@ const EnhancedTableToolbar = ({
           />
         </div>
         <ul className="flex flex-wrap gap-2 xs:flex-nowrap">
-          {["all", "deposited", "stable", "volatile"].map((filterOption) => (
-            <li key={filterOption}>
-              <input
-                type="radio"
-                id={filterOption}
-                name="filter"
-                value={filterOption}
-                checked={filter === filterOption}
-                onChange={onChange}
-                className="peer hidden"
-              />
-              <label
-                htmlFor={filterOption}
-                className="flex min-h-[56px] min-w-[108px] cursor-pointer items-center justify-center rounded-lg border border-[rgba(255,255,255,0.23)] px-2 font-medium transition-colors hover:bg-emerald-900 peer-checked:border-emerald-900 peer-checked:bg-background peer-checked:font-semibold peer-checked:text-lime-50"
-              >
-                <div className="uppercase">{filterOption}</div>
-              </label>
-            </li>
-          ))}
+          {["all", "boosted", "deposited", "stable", "volatile"].map(
+            (filterOption) => (
+              <li key={filterOption}>
+                <input
+                  type="radio"
+                  id={filterOption}
+                  name="filter"
+                  value={filterOption}
+                  checked={filter === filterOption}
+                  onChange={onChange}
+                  className="peer hidden"
+                />
+                <label
+                  htmlFor={filterOption}
+                  className="flex min-h-[56px] min-w-[108px] cursor-pointer items-center justify-center rounded-lg border border-[rgba(255,255,255,0.23)] px-2 font-medium transition-colors hover:bg-emerald-900 peer-checked:border-emerald-900 peer-checked:bg-background peer-checked:font-semibold peer-checked:text-lime-50"
+                >
+                  <div className="uppercase">{filterOption}</div>
```

Comment:
```suggestion
                  <span className="capitalize">{filterOption}</span>
```
Why not use `span`? And have you tried `capitalize`?

<img width="645" alt="image" src="https://github.com/Velocimeter/frontend/assets/126733611/3be48cd0-24dd-4ac6-b7e5-e4b4e9746645">

Much delicate IMO.

Reply:
we can come up with something but i like uppercase the most in this case, and italic the least